### PR TITLE
Add an example notebook for training using low-priority AML compute

### PR DESCRIPTION
**Pull request by daholste**
# Add an example notebook for training using low-priority AML compute

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

- Add an example notebook for training using low-priority AML compute

### CODE REVIEW

File: python-sdk/README.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ path|status|notebooks|description
...
 [deploy-local](tutorials/deploy-local)|[![deploy-local](https://github.com/Azure/azureml-examples/workflows/python-sdk-tutorial-deploy-local/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/python-sdk-tutorial-deploy-local.yml)|[1.deploy-local.ipynb](tutorials/deploy-local/1.deploy-local.ipynb)<br>[2.deploy-local-cli.ipynb](tutorials/deploy-local/2.deploy-local-cli.ipynb)|*no description*
+[train-on-low-priority-aml-compute](tutorials/train-on-low-priority-aml-compute)|[![train-on-low-priority-aml-compute](https://github.com/Azure/azureml-examples/workflows/train-on-low-priority-aml-compute/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/python-sdk-tutorial-train-on-low-priority-aml-compute.yml)|[train-on-low-priority-aml-compute.ipynb](tutorials/train-on-low-priority-aml-compute/train-on-low-priority-aml-compute.ipynb)|Learn how to train models using low-priority AML compute.
```
Comment (jeff-shepherd) [inline python-sdk/README.md:61]: You could add some explanation of what low-priority AML compute is and why it is useful.
Something like:
"Low-priority compute costs significantly less than dedicate compute but jobs on low priority compute can be preempted"

File: python-sdk/tutorials/train-on-low-priority-aml-compute/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- python
+- azurecli
+products:
+- azure-machine-learning
+description: Learn how to train models using low-priority AML compute.
+---
+
+# Using Low-Priority AML Compute
+
+Learn how to train models using low-priority AML compute.
```
Comment (jeff-shepherd) [inline python-sdk/tutorials/train-on-low-priority-aml-compute/README.md:13]: You could add some description of what this example does.
For example,
"The example uses Pytorch to train a model to recognize handwritten digits."

**Review by jeff-shepherd — APPROVED** [review]

**Review by swatig007 — APPROVED** [review]

File: python-sdk/tutorials/train-on-low-priority-aml-compute/README.md (hunk 2 above)
Comment (daholste) [reply python-sdk/tutorials/train-on-low-priority-aml-compute/README.md]: Thank you! I used this verbatim!

File: python-sdk/README.md (hunk 1 above)
Comment (daholste) [reply python-sdk/README.md]: Thank you! I used this verbatim!

**Pull request merged**